app/signup/page.tsx

```
@@ -1,44 +1,64 @@
     }
-  }, [user, router])
+  }, [router, isClient])
+
+  // Don't render anything until client is mounted or if bypassing
+  if (!isClient || BYPASS_CLERK) {
+    return null
+  }
 
+  // From here on, we know we're on the client and not bypassing, so we can safely import Clerk
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const { useSignUp, useUser } = require("@clerk/nextjs")
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)